### ISSUE TITLE
Resync with instance file

### ISSUE THREAD

**Issue by vr-fox** [thtliife/servicenow-sync#8]
Is there a way to re-retrive the file from the instance... 3 persons are working in the same file, how to resync with the file that is in the instance instead of just uploading

**Comment by thtliife** [comment]
In the config panel, you can pull the existing content from an instance. (See example pic)

![Retrieve from instance](https://cloud.githubusercontent.com/assets/9210782/24273288/304e6bd2-106d-11e7-9749-caa06b4e9bdb.png)

Next version of the plugin will be a full rewrite, but will include this functionality exposed as an actual command in the command palette, which means you could also create a key binding for it.
In the current version however, that is not exposed, and the functionality is only available from the config panel (default `ctrl+alt+a`)

**Comment by vr-fox** [comment]
Oops i didn't saw all fileds where filled. Thx. Also i will do another pull request, i just added support for  ui actions

**Comment by thtliife** [comment]
@vr-fox 
Just a quick FYI.
You can put a file named `tableConfig.user.cson` in the folder at `~/.atom/packages/servicenow-sync/config` (Mac/Linux) or `C:\Users\username\.atom\packages\servicenow-sync\config` (Windows).
The servicenow-sync package will read that and merge its contents with the default `tableConfig.cson`.

It must be formatted in the same manner as the default `tableConfig.cson` is...

So there really isn't a huge need to keep submitting pr's for new table support. Although for default tables introduced in geneva/helsinki I think it is still a good idea to include them as default. (I wrote this package when the fuji was the current instance.)

**Comment by vr-fox** [comment]
Yeah i saw that file, but as you said, this are default tables, and for my team it is easier when they just install the package from atom and not me having to send them the user config. But thanks as i use it for not default tables
